app/forms/nqt_in_academic_year_after_itt_form.rb

```diff
@@ -0,0 +1,65 @@
+class NqtInAcademicYearAfterIttForm < Form
```

Comment:
I namespaced `SupplyTeacherForm` under `Journeys:AdditionalPaymentsForTeaching` since it's journey specific.

Reply:
Makes sense. Will reissue this pr with name spacing sorted once I've rebased it on Giuseppe's work